### ISSUE TITLE
precision

### ISSUE THREAD

**Issue by hwddean** [kachiO/class-agnostic-counting-pytorch#2]
I'd like to ask two questions.
1、Is it possible to reproduce the accuracy of the paper?
2、Can you count classes that have not been trained?
